functions/auth.bash

```diff
@@ -98,3 +98,38 @@ change_password() {
     whiptail --title "Operation Successful!" --msgbox "Password(s) successfully changed for: $chosenAccounts" 8 80
   fi
 }
+
+
+## Function to download SSH key pair for remote access
+## 
+## The function can be invoked during UNATTENDED installation only.
+## It downloads a ssh key from a user specified location and allows the key owner to login as the admin user
+##
+##    add_admin_ssh_key()
+##
+add_admin_ssh_key() {
```

Comment:
Add a check to ensure that tailscale is installed before running.

Reply:
I'd think that is not necessarily related ? You could want not to use tailscale but say port forwarding and use the admin key nonetheless.

Reply:
But this function directly calls tailscale cli if I recall correctly, right?

Reply:
duh, right. I moved adding the key to console to setup_tailscale()